projects/loopback.json

```diff
@@ -159,6 +159,62 @@
     "strongloop/strong-error-handler",
     "strongloop/strong-pubsub",
     "strongloop/strong-remoting",
-    "strongloop/strong-soap"
+    "strongloop/strong-soap",
+    "strongloop-community/loopback-connector-redis", 
+    "strongloop-community/loopback-connector-elastic-search", 
+    "strongloop-community/loopback-connector-mysql-jdbc",
+    "strongloop-community/loopback-benchmark-mongodb",
+    "strongloop-community/loopback-acl-route", 
+    "strongloop-community/loopback-connector-sqlite3", 
```

Comment:
i think we only need:
- loopback-connector-redis
- loopback-connector-sqlite3
for now. We can add manually for the other repos that need in the future.